### PR TITLE
WIP Translation first steps

### PR DESCRIPTION
**Pull request by dragonito**

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -9,14 +9,17 @@ fos_user:
...
+        fallback:   en
+        logging:    true
     secret:          %secret%
     router:          { resource: "%kernel.root_dir%/config/routing.yml" }
     form:            true
     csrf_protection: true
     validation:      { enable_annotations: true }
     templating:      { engines: ['twig'] } #assets_version: SomeVersionScheme
-    default_locale:  %locale%
+    default_locale:  de
```
Comment (tolry) [inline app/config/config.yml]: why not keep using %locale% from parameters.yml?

Comment (DavidBadura) [inline app/config/config.yml]: :+1:

Comment (dragonito) [inline app/config/config.yml]: I just get it from the Symfony documentation. What does %locale% ?

Comment (tolry) [inline app/config/config.yml]: %locale% resolves to whatever you have configured for "locale" in your app/config/parameters.yml

**Comment by tolry** [comment]
PR closed by @dragonito's request

**Pull request closed without merging**